ts/screens/preferences/ServiceDetailsScreen.tsx

```diff
@@ -319,11 +313,7 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
               </Col>
             </Row>
             <View spacer={true} large={true} />
-            {description && (
-              <Markdown lazyOptions={{ lazy: true, animated: true }}>
-                {description}
-              </Markdown>
-            )}
+            {description && <Markdown>{description}</Markdown>}
```

Comment:
we probably need to add the `animated` props as well, it used to animate the layout of the Markdown component in the service detail screen, as there's some content after the markdown - can you please check that with the current component the layout doesn't change abruptly ?